apps/client/src/app/pages/portfolio/transactions/transactions-page.component.ts

```diff
@@ -204,8 +203,8 @@ export class TransactionsPageComponent implements OnDestroy, OnInit {
           } else if (file.name.endsWith('.csv')) {
             try {
               await this.importTransactionsService.importCsv({
+                user: this.user,
```

Comment:
Can you move `user` down? For consistency, we are ordering attributes alphabetically 🙂

Reply:
now i'm aware